From 67e7d350a15aff88c151b1fc838dac83d35be955 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd@kuhls.net>
Date: Sat, 22 Nov 2025 23:47:41 +0100
Subject: [PATCH] atf-check.cpp: include time.h

Fixes build error with gcc 14 as reported by the buildroot autobuilders:
https://autobuild.buildroot.net/results/41b/41b25ee8e66e34323eca011e4b5fe479ece9ed76/build-end.log

atf-sh/atf-check.cpp: In function 'useconds_t get_monotonic_useconds()':
atf-sh/atf-check.cpp:183:24: error: 'CLOCK_MONOTONIC' was not declared in this scope

Upstream: https://github.com/freebsd/atf/commit/67e7d350a15aff88c151b1fc838dac83d35be955

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 atf-sh/atf-check.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/atf-sh/atf-check.cpp b/atf-sh/atf-check.cpp
index 1354e3a..94da413 100644
--- a/atf-sh/atf-check.cpp
+++ b/atf-sh/atf-check.cpp
@@ -30,6 +30,7 @@ extern "C" {
 #include <limits.h>
 #include <signal.h>
 #include <stdint.h>
+#include <time.h>
 #include <unistd.h>
 }
 
-- 
2.47.3

